### PR TITLE
Checkin components, styles and docs

### PR DESCRIPTION
**Pull request by j1mie**
Adds ICS React components, styles, icons, and docs. 

#### Changelog

**New**

- ICS React components, tests, stories and styles
- Global styles including grid, type, layer, icons and bidi
- Documentation

**Changed**

- Updates Readme to include usage info

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: node_js
+node_js: 6
+group: bluezone
```
Comment (joshblack) [inline .travis.yml]: Unnecessary for public TravisCI, right?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+install:
+  - npm install
+script:
+  - npm run ci-check
+before_deploy:
```
Comment (joshblack) [inline .travis.yml]: Want to omit this block and I'll add it in with the TravisCI update for build/deploy/publishing?

File: .travis.yml (hunk 1 above)
Comment (j1mie) [reply .travis.yml]: @joshblack True that. Will update.

File: .gitignore [hunk 3]
```diff
@@ -1,20 +1,29 @@
...
 
-# Generated by npm@5, but project currently uses Yarn
+# we yarn folks here
 package-lock.json
+
+# IDE
+.idea/
+
+# jest
+__snapshots__
```
Comment (joshblack) [inline .gitignore]: Any reason to ignore snapshot files?

File: .gitignore [hunk 4]
```diff
@@ -1,20 +1,29 @@
...
+es
+storybook-static
+/scss
+
+# a11y files
+a11y/results
 
-# Generated by npm@5, but project currently uses Yarn
+# we yarn folks here
 package-lock.json
```
Comment (joshblack) [inline .gitignore]: Are you all using Yarn or npm? If npm, should the lockfile be ignored?

Comment (j1mie) [reply .gitignore]: @joshblack Using npm - will remove this from gitignore

**Comment by j1mie** [comment]
Cheers Josh! 👏

**Pull request merged**